client/update_test.go

```diff
@@ -42,3 +43,19 @@ func TestChannel_Update_NilArgs(t *testing.T) {
 	require.Error(t, err)
 	assert.Contains(t, err.Error(), "context")
 }
+
+func TestRequestTimedOutError(t *testing.T) {
+	var err error = errors.WithStack(newRequestTimedOutError("", ""))
```

Comment:
It should already have a stacktrace now.

Reply:
Yeah, i missed to remove `WithStack` after adding the `newRequestTimedOutError` in previous review iteration. It have updated it.